workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/learn-search-capability/page-definition.json

```diff
@@ -0,0 +1,598 @@
+{
+	"pageElement": {
+		"id": "5aacd162-daab-4a5a-2a14-a2389d528087",
+		"pageElements": [
+			{
+				"definition": {
+					"fragment": {
+						"key": "learn-capability-search-page",
+						"siteKey": "Liferay Learn"
+					},
+					"fragmentConfig": {
+					},
+					"fragmentFields": [
+					],
+					"indexed": true
+				},
+				"id": "da6953a4-d056-277f-be2d-c53f74c8cc52",
+				"pageElements": [
+					{
+						"definition": {
+						},
+						"id": "be6050fc-dca5-1dd5-d209-8b6be31f3718",
+						"pageElements": [
+							{
+								"definition": {
+									"cssClasses": [
+										"learn-banner"
+									],
+									"indexed": true,
+									"layout": {
+									},
+									"name": "Banner Container"
+								},
+								"id": "6a130234-4939-8781-572b-220a62eb93e0",
+								"pageElements": [
+									{
+										"definition": {
+											"fragmentStyle": {
+												"maxWidth": "var(--container-max-xl)",
+												"paddingBottom": "5",
+												"paddingLeft": "4",
+												"paddingRight": "4",
+												"paddingTop": "5",
+												"width": "100%"
+											},
+											"indexed": true,
+											"layout": {
+												"paddingBottom": 4,
+												"paddingLeft": 2,
+												"paddingRight": 2,
+												"paddingTop": 4,
+												"widthType": "Fixed"
+											}
+										},
+										"id": "b034a4fa-2c79-d945-037f-681c9363d661",
+										"pageElements": [
+											{
+												"definition": {
+													"fragmentStyle": {
+														"maxWidth": "var(--container-max-sm)",
+														"textColor": ""
+													},
+													"indexed": true,
+													"layout": {
+													}
+												},
+												"id": "a48c452e-eecb-ba53-db44-ff8e2489e549",
+												"pageElements": [
+													{
+														"definition": {
+															"fragment": {
+																"key": "dialect-text",
+																"siteKey": "Liferay Learn"
+															},
+															"fragmentConfig": {
+																"textType": "text-paragraph-lg",
+																"truncateText": false
+															},
+															"fragmentFields": [
+																{
+																	"id": "element-text",
+																	"value": {
+																		"fragmentLink": {
+																		},
+																		"text": {
+																			"mapping": {
+																				"fieldKey": "AssetCategory_vocabulary",
+																				"itemReference": {
+																					"className": "com.liferay.asset.kernel.model.AssetCategory",
+																					"classPK": 32495601
```

Comment:
@luisacmn 
It's necessary to put a placeholder to do a replace.
Please use TAXONOMY_CATEGORY_ID as placeholder for all classPK.